src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int maxFigureLength = 7;
```

Comment:
use CONSTANT_CASE for naming

Reply:
![image](https://github.com/user-attachments/assets/f4998f05-6220-401e-88ad-8fb099db126f)

if i do that i will fail my tests